add-chain/testdata/superchain/configs/sepolia/expected_zorasep.toml

```diff
@@ -8,7 +8,9 @@ standard_chain_candidate = true # This is a temporary field which causes most of
 batch_inbox_addr = "0xCd734290E4bd0200dAC631c7D4b9E8a33234e91f"
 block_time = 2
 seq_window_size = 3600
+max_sequencer_drift = 600
 data_availability_type = "eth-da"
+channel_timeout_bedrock = 300
```

Comment:
out of curiosity, why are we not including granite channel timeout here and in others configs that follow?

Reply:
For chains that haven't activated the `granite` hardfork yet, the `channel_timeout_granite` setting is not yet applicable. In an effort to keep the config files clean, we typically exclude fields that are not applicable. If/when `granite` is activated for a given chain, we should add that field to the chain's config file.